Library/Formula/erlang.rb

```diff
@@ -39,11 +39,13 @@ class Erlang < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
-  depends_on "openssl"
   depends_on "unixodbc" if MacOS.version >= :mavericks
   depends_on "fop" => :optional # enables building PDF docs
   depends_on "wxmac" => :recommended # for GUI apps like observer
 
+  depends_on "libressl" => :optional
+  depends_on "openssl" unless build.with? "libressl"
```

Comment:
```
depends_on "openssl" => :recommended
depends_on "libressl" => :optional
```

Has been the accepted syntax for this use-case thus far. Because of the specification of the ssl prefix in the ` args ` Homebrew won't install with OpenSSL even if you don't use the second flag. We're also trying to move away from ` if build.without? ` deps, as they're prone to breakage.